rom/dev/src/main.rs

```diff
@@ -53,6 +54,13 @@ Running Caliptra ROM ...
 pub extern "C" fn rom_entry() -> ! {
     cprintln!("{}", BANNER);
 
+    if !cfg!(feature = "no-cfi") {
```

Comment:
in other places this feature is called "no_cfi", here it is "no-cfi"

Reply:
Typo; fixed.